tests/pytest_tests/unit_tests/test_lib/test_paths.py

```diff
@@ -32,6 +32,7 @@ def test_path_groups(target, posix, windows, _bytes):
 @example("/")
 @example(PureWindowsPath("C:/foo/bar.txt"))
 @example(PurePosixPath("x\\a/b.bin"))
+@pytest.mark.xfail(reason="TODO: Hypothesis keeps finding corner cases.")
```

Comment:
seems like this test need to be refined or if these are real cases the underlying logic should be fixed.
it feels wrong to xfail it... if that's the solution we could just skip it